quickstarts/dapr/config.yml

```diff
@@ -0,0 +1,27 @@
+id: 658d36f3-aa96-4e74-93da-b74eefcaabd3
```

Comment:
New quickstarts will have their id generated after their first merge, lets remove this field

Reply:
id removed